### PR TITLE
build: create Features of Balanty

### PR DESCRIPTION
**Pull request by Heba-Adwan**
create Features of Balanty #27

### CODE REVIEW

**Review by Emanalabsi — APPROVED** [review]

File: client/src/components/Features.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import feature from '../assets/feature.svg';
+
+import {
+  StyledBoxFeatures,
+  StyledImageFeatures,
+  StyledDescriptionFeatures,
+  StyledTitle,
+} from './index';
```
Comment (alaaahmade) [inline client/src/components/Features.tsx:9]: There is nothing exported from index.ts

File: client/src/components/Features.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+const FeatureSection: React.FC<FeatureProps> = ({ imageSrc, description }) => {
+  return (
+    <StyledBoxFeatures>
+      <StyledImageFeatures src={imageSrc} alt="" />
+      <StyledDescriptionFeatures>{description}</StyledDescriptionFeatures>
+    </StyledBoxFeatures>
+  );
+};
+
```
Comment (alaaahmade) [inline client/src/components/Features.tsx:24]: Separate every React Component with a special file

File: client/src/components/styledRootComponent/FeaturesStyledComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import styled from '@emotion/styled';
+
+export const StyledBoxFeatures = styled.div`
```
Comment (alaaahmade) [inline client/src/components/styledRootComponent/FeaturesStyledComponent.tsx:3]: We want to use the styled component provided by Mui

**Review by alaaahmade — CHANGES_REQUESTED** [review]

File: client/src/components/Features.tsx [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  StyledDescriptionFeatures,
+  StyledTitle,
+} from './index';
+
+interface FeatureProps {
+  imageSrc: string;
+  description: string;
+}
+
+const FeatureSection: React.FC<FeatureProps> = ({ imageSrc, description }) => {
```
Comment (alaaahmade) [inline client/src/components/Features.tsx:16]: done forget to add the type of returning function : ReactElement

File: client/src/components/Features.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  return (
+    <StyledBoxFeatures>
+      <StyledImageFeatures src={imageSrc} alt="" />
+      <StyledDescriptionFeatures>{description}</StyledDescriptionFeatures>
+    </StyledBoxFeatures>
+  );
+};
+
+const Feature: React.FC = () => {
```
Comment (alaaahmade) [inline client/src/components/Features.tsx:25]: done forget to add the type of returning function : ReactElement

**Review by alaaahmade — CHANGES_REQUESTED** [review]

File: client/src/components/Features.tsx (hunk 4 above)
Comment (alaaahmade) [inline client/src/components/Features.tsx:16]: Separate every React Component with a special file

**Review by alaaahmade — CHANGES_REQUESTED** [review]

**Review by Saeed99Madi — APPROVED** [review]

**Review by ahmedirheem — APPROVED** [review]

**Pull request merged**